ipywidgets/widgets/widget_string.py

```diff
@@ -40,6 +40,19 @@ class HTML(_String):
     _view_name = Unicode('HTMLView').tag(sync=True)
     _model_name = Unicode('HTMLModel').tag(sync=True)
 
+
+@register
+class HTMLElement(_String):
```

Comment:
I was not speaking about a new widget... I was speaking about https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement, or https://developer.mozilla.org/en-US/docs/Web/API/HTMLDivElement.